src/biophysics.jl

```diff
@@ -370,4 +370,123 @@ function evap(
   J_evap = uconvert(u"g/s",J_evap)
 
   (Q_evap = Q_evap, J_evap = J_evap, J_resp = J_resp, J_cut = J_cut, J_eyes = J_eyes)
+end
+
+function resp(
+    T_x = (23+273.15)K,
```

Comment:
```suggestion
    T_x = K(23°C),
```

I think this is clearer?

Reply:
It wont be type unstable (julia will work that out) but it might be slow. So yes `23.0` may be better?

The problem with using `23.0` is e.g. on a GPU you may want to use all `Float32` (`Float64` can be much slower) but using `23.0` will bump the whole equation up to `Float64`.

We don't have to get it perfect the first time but it's something to think about. Using the `zero`, `one` and `oneunit` functions can help get around these issues.